src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -545,109 +554,43 @@ public sealed partial class SqlConnectionStringBuilder : System.Data.Common.DbCo
-        public static bool Start(string connectionString, string queue) { throw null; }
-        public static bool Stop(string connectionString) { throw null; }
-        public static bool Stop(string connectionString, string queue) { throw null; }
-    }
-    public delegate void OnChangeEventHandler(object sender, SqlNotificationEventArgs e);
-    public partial class SqlNotificationEventArgs : System.EventArgs
-    {
-        public SqlNotificationEventArgs(SqlNotificationType type, SqlNotificationInfo info, SqlNotificationSource source) { }
-        public SqlNotificationType Type { get { throw null; } }
-        public SqlNotificationInfo Info { get { throw null; } }
-        public SqlNotificationSource Source { get { throw null; } }
-    }
-    public enum SqlNotificationInfo
-    {
-        Truncate = 0,
-        Insert = 1,
-        Update = 2,
-        Delete = 3,
-        Drop = 4,
-        Alter = 5,
-        Restart = 6,
-        Error = 7,
-        Query = 8,
-        Invalid = 9,
-        Options = 10,
-        Isolation = 11,
-        Expired = 12,
-        Resource = 13,
-        PreviousFire = 14,
-        TemplateLimit = 15,
-        Merge = 16,
-        Unknown = -1,
-        AlreadyChanged = -2
-    }
-    public enum SqlNotificationSource
-    {
-        Data = 0,
-        Timeout = 1,
-        Object = 2,
-        Database = 3,
-        System = 4,
-        Statement = 5,
-        Environment = 6,
-        Execution = 7,
-        Owner = 8,
-        Unknown = -1,
-        Client = -2
-    }
-    public enum SqlNotificationType
-    {
-        Change = 0,
-        Subscribe = 1,
-        Unknown = -1
-    }
-    public sealed partial class SqlRowUpdatedEventArgs : System.Data.Common.RowUpdatedEventArgs
-    {
-        public SqlRowUpdatedEventArgs(DataRow row, IDbCommand command, StatementType statementType, System.Data.Common.DataTableMapping tableMapping)
-        : base(row, command, statementType, tableMapping) { }
-
-        new public SqlCommand Command { get { throw null; } }
-    }
-    public delegate void SqlRowUpdatedEventHandler(object sender, SqlRowUpdatedEventArgs e);
-    public sealed partial class SqlRowUpdatingEventArgs : System.Data.Common.RowUpdatingEventArgs
-    {
-        public SqlRowUpdatingEventArgs(DataRow row, IDbCommand command, StatementType statementType, System.Data.Common.DataTableMapping tableMapping)
-        : base(row, command, statementType, tableMapping) { }
-
-        new public SqlCommand Command { get { throw null; } set { } }
-        protected override System.Data.IDbCommand BaseCommand { get { throw null; } set { } }
-    }
-    public delegate void SqlRowUpdatingEventHandler(object sender, SqlRowUpdatingEventArgs e);
-    public partial class SqlDataReader : System.Data.Common.DbDataReader, System.IDisposable
+    public partial class SqlDataReader : System.Data.Common.DbDataReader, System.Data.IDataReader, System.Data.IDataRecord, System.IDisposable
```

Comment:
Is this the expected type signature? I added more interface implementations (`System.Data.IDataReader`, `System.Data.IDataRecord`)